commit ac0a3e61aa3a7005b62e3b64f238381370a5e458 (refs/changes/34/4534/1)
Author: Rui Loura <rui@joyent.com>
Date:   2018-07-18T15:54:37+00:00 (1 year, 3 months ago)
    
    TRITON-606 ur-agent should be aware of alternate admin nictags

diff --git a/ur-agent b/ur-agent
index be2c4c7..9a3dee0 100644
--- a/ur-agent
+++ b/ur-agent
@@ -247,7 +247,11 @@ function UrAgent(options) {
     function firstAdminMac() {
         var interfaces;
         var mac;
+        var admin_tag = 'admin';
 
+        if (self.sysinfo['Admin NIC Tag']) {
+            admin_tag = self.sysinfo['Admin NIC Tag'];
+        }
         interfaces = self.sysinfo['Network Interfaces'];
 
         for (var iface in interfaces) {
@@ -256,7 +260,7 @@ function UrAgent(options) {
             }
 
             var nic = interfaces[iface]['NIC Names'];
-            var isAdmin = nic.indexOf('admin') !== -1;
+            var isAdmin = nic.indexOf(admin_tag) !== -1;
             if (isAdmin) {
                 mac = interfaces[iface]['MAC Address'];
                 return mac;
